index.html

```diff
@@ -4,162 +4,89 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <meta name="author" content="Choco Li" />
-    <meta name="description" content="2 full page demonstration" />
-    <meta name="keywords" content="orchard, healthy, food" />
-    <meta name="Resource-type" content="Document" />
     <title>Orchard</title>
   </head>
-  <body>
-    <div id="fullpage">
-      <div class="section active" id="section0">
-        <section class="story">
-          <div class="content content--x-center">
-            <div class="imageBox">
-              <img
-                class="imageBox__image"
-                src="./img/crossField.png"
-                alt="a man carrying a child crossing a crop field"
-              />
+  <body class="bgBlack">
+    <div class="container">
+      <section class="row my-15 align-items-center">
+        <div class="px07 col-lg-4 col-md-6">
+          <img src="./img/crossField.png" alt="crossField" class="mw-100" />
+        </div>
+        <div class="px07 col-lg-4 col-md-6 d-flex flex-column">
+          <img src="./img/fruitInPlate.png" alt="fruit" class="mw-100 py07" />
+          <img src="./img/asparagus.png" alt="asparagus" class="mw-100 py07" />
+        </div>
+        <article class="px07 col-lg-4 col-md-12 textWhite align-self-start">
+          <h4 class="border-bottom py-3 textSpacing_04">
+            <a href="">ANSWER YOUR BODY'S NEEDS</a>
+          </h4>
+          <p class="py-4 fontSize16 lineHeight28">
+            The way ingredients are sourced affects the way we nourish our
+            bodies. Author Mark Schatzer believes our body naturally devolops an
+            appetite for the foods and nutrients it needs to be healthy, but
+            that artificial flavourings are getting in the way. This can be
+            reversed by focusing on high-quality ingredients and being mindful
+            as your appetite guides you to consume according to your body's
+            needs.
+          </p>
+          <h6 class="textRed textBold">BE MINDFUL</h6>
```

Comment:
Quite minor. You can use `text-transform: uppercase` rather than doing it in the HTML.

Reply:
Sure, I am happy to add `text-uppercase` class.